rosbag2_cpp/test/rosbag2_cpp/test_sequential_reader.cpp

```diff
@@ -221,14 +222,22 @@ TEST_F(SequentialReaderTest, next_file_calls_callback) {
   EXPECT_EQ(opened_file, bag_file_2_path_.string());
 }
 
-TEST_F(TemporaryDirectoryFixture, reader_accepts_bare_file) {
+class ParametrizedTemporaryDirectoryFixture : public TemporaryDirectoryFixture,
```

Comment:
Move this class to `temporary_directory_fixture` instead of copy-paste in these tests? Call it `StorageIdParamFixture` or something.